### ISSUE TITLE
Examples: Grep

### ISSUE THREAD

**Issue by sirupsen** [danlucraft/rak#1]
I recall in `grep` you can do:

```
$ grep -r pattern
```

Rather than:

```
$ grep pattern $(find . | grep -v .svn)
```

As shown in the examples -- or am I getting something wrong here?

**Comment by danlucraft** [comment]
Yes. But the first one wouldn't exclude svn directories, would it?

**Comment by sirupsen** [comment]
Ah -- that might be right -- but who uses SVN anyway?

On a serious note: 
    --exclude-dir=.svn

**Comment by danlucraft** [comment]
I don't have that option in my grep. But I think I can do --exclude=.svn, possibly.

**Comment by sirupsen** [comment]
Possibly -- I'm running `grep` 2.6.3.

I'm sure `--exclude=.svn` would work as well; `--excude-dir` was just more explicit. :)

**Comment by taw** [comment]
`grep pattern (find . | grep -v .svn)` won't really work:
- It will die with "Argument list too long" if you're grepping even moderately sized source tree. ARGV limits are puny - 128kB on Linux, 256kB on OSX.
- It will also fail if you have any funny characters in your paths, where even space is considered funny.
